frontend/javascripts/admin/admin_rest_api.js

```diff
@@ -1396,6 +1396,23 @@ export async function checkAnyOrganizationExists(): Promise<boolean> {
   return !(await Request.receiveJSON("/api/organizationsIsEmpty"));
 }
 
+export async function deleteOrganization(organizationName: string): Promise<void> {
+  return Request.triggerRequest(`/api/organizations/${organizationName}`, {
+    method: "DELETE",
+  });
+}
+
+export async function updateOrganization(
+  organizationName: string,
+  displayName: string,
+  newUserMailingList: string,
+): Promise<void> {
```

Comment:
Are you sure that this does not return a `Promise<APIOrganization>` or something like this? Didn't test it, but I think that's often what the backend is doing..

Reply:
Yes, the backend returns the updated organization as json again :+1: